scripts/copy_emmc.sh

```diff
@@ -0,0 +1,7 @@
+# Script to copy the root partition from SDcard to eMMC memory
+# date 08/02/2024
+# author: Matin
+
+#!/bin/bash
+
```

Comment:
it would be good to include the mount commands as well to specify the disk mounted to `/mnt`